.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ jobs:
           name: utils
           command: |
             whoami
-            curl https://cli-assets.heroku.com/apt/release.key | sudo apt-key add -
+            # curl https://cli-assets.heroku.com/apt/release.key | sudo apt-key add -
```

Comment:
Why commented out?

Applies to the below changes.

Reply:
I removed this commented line as its not been used in any place.